src/main/java/desktop/pages/LoginPage.java

```diff
@@ -0,0 +1,65 @@
+package desktop.pages;
+
+import abstractclasses.page.AbstractPage;
+import desktop.fragments.MainHeader;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+
+public class LoginPage extends AbstractPage {
+
+    private static final String PLACEHOLDER = "placeholder";
+
+    @FindBy(id = "ap_customer_name")
+    private WebElement name;
```

Comment:
good practice is to add element's type to the end of WebElement's name - e.g. nameInput. after this you can remove "this." in your methods, so there wouldn't be any conflicts between method's argument and WebElement

Reply:
done